### ISSUE TITLE
Resolve accessibility regressions

### ISSUE THREAD

**Issue by TimBoettcher** [DavidKinder/Windows-Inform7#46]
This might be related to the following issue: #27 

I detected the following accessibility issues when attempting to use Windows Inform 7 Beta 5:

### JAWS (Version: 2021.2111.13)

Many panels, such as the report page (on release and compilation outcome) and the panel the game gets played in appear to be completely empty when using a braille display with JAWS' standard cursor. As opposed to the last non-beta release of Windows Inform 7, the used cursor in the report page is the "PC cursor", not the "virtual PC cursor", indicating that JAWS does not perceive the panel as a HTML document anymore and instead tries treating it like a native Windows application, which apparently fails - at any rate, I cannot detect any text, tab to any controls.
Using the JAWS cursor (tethered to the mouse pointer) I can only perceive the menu and tab panels, but not the content of the panel. Using the touch cursor I can perceive the content of the panel, but navigation is tedious and in the game panel, the game's content is represented as a single line, forcing me to scroll far with the braille panning buttons if the game session is lengthy; also, all information on formatting, such as paragraphs, is lost.

### NVDA (Version: 2021.2)

The game panel is accessible, but the report panel is not - neither for reports on compilation, nor for reports on release. There appears to be no workaround to perceive the content of the panels. I also occasionally experienced issues with the welcome screen where the content of that screen was invisible, though that problem did not occur consistently.

### Narrator (on Windows 10)

Narrator performs best in many ways, all panels but one appear to get represented correctly. Unfortuantely, that one panel is the source panel - the source text is entirely invisible, and nothing I tried could change that fact (albeit I admit that I'm not too familiar with Narrator, so this might be a problem caused by my usage and not Windows Inform 7).

If you have any further questions or would like me to test further, let me know.€€

**Comment by DavidKinder** [comment]
Also see https://intfiction.org/t/inform-7-for-windows-accessibility-observations/69353
